client/src/components/Auth/PrivateRoute/__tests__/PrivateRoute.test.jsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import { MemoryRouter } from "react-router";
+import { mount } from "enzyme";
+import { findByTestAttr } from "../../../../test/testUtils";
+import { GenericComponent } from "../../../../test/__mocks__/GenericComponent";
+import PrivateRoute from "../PrivateRoute";
+import thunk from "redux-thunk";
+import configureStore from "redux-mock-store";
+
+const mockStore = configureStore([thunk]);
```

Comment:
It's still probably useful to not have to call `configureStore([thunk])` within every individual test. You don't need to create a wrapper around `mockStore`, but creating `mockStore` once would be helpful.